src/common/utility_win.cpp

```diff
@@ -234,8 +234,8 @@ void Utility::removeFavLink(const QString &folder)
 
     // #2 Remove the system file attribute
     const auto folderAttrs = GetFileAttributesW(folder.toStdWString().c_str());
-    if (!SetFileAttributesW(folder.toStdWString().c_str(), folderAttrs & ~FILE_ATTRIBUTE_SYSTEM)) {
-        qCWarning(lcUtility) << "Remove system file attribute failed for:" << folder;
+    if (!SetFileAttributesW(folder.toStdWString().c_str(), folderAttrs & ~FILE_ATTRIBUTE_READONLY)) {
```

Comment:
what would happen if this folder was using the system flags ?
we would need to care about existing users